src/psrsearch.jl

```diff
@@ -159,12 +155,12 @@ function z_n_search(
     t0 = first(times)
     t1 = last(times)
     df = 1 / (t1 - t0) / oversample
-    freqs = [fmin:df:fmax;]
+    freqs = fmin:df:fmax
     N = length(freqs)
-    stats = zeros(N)
-    for i in range(1, stop = N)
+    stats = Vector{T}(undef, N)
```

Comment:
In the `for` loop below you're going to set all the elements of `stats` anyway: you can shave off some microseconds by initialising to garbage memory instead of setting to zeros.